test/worldwide/config_test.rb

```diff
@@ -72,7 +72,7 @@ class ConfigTest < ActiveSupport::TestCase
 
     test "#configure applies changes to I18n.config by default" do
       I18n.config.expects(:enforce_available_locales=).never
-      I18n.config.expects(:load_path=).times(4)
+      I18n.config.expects(:load_path=).times(3)
 
       assert_instance_of I18n::Config, Worldwide::Config.configure_i18n
     end
```

Comment:
Nit: brittle test. Perhaps instead of expecting the call, we can expect a change in the `I18n.load_path.size`?